docs/metatype.dev/blog/2024-09-26-introducing-grpc-runtime/index.mdx

```diff
@@ -0,0 +1,92 @@
+import BlogIntro from "@site/src/components/BlogIntro";
+
+# Introducing gRPC Runtime
+
+<BlogIntro text="We're excited to announce the new gRPC Runtime feature in Metatype, further enhancing our platform's ability to create versatile and powerful backends through typegraphs." />
+
+## What is Metatype?
+
+Before we dive into the gRPC Runtime, let's recap what Metatype is all about:
+
+Metatype is a platform that allows developers to create typegraphs - strongly-typed, composable backend structures. With Metatype, you can:
+
+- Construct solid, type-safe backends
+- Leverage multiple runtimes for different services and protocols
+- Expose your typegraphs via GraphQL or HTTP endpoints
```

Comment:
link to more usecases? I am not sure we should speak about metatype on the first paragraph, rather at the end when the concrete example comes. Look at the temporal blog article